waspc/package.yaml

```diff
@@ -64,6 +68,8 @@ library:
     - utf8-string
     - Glob
     - unliftio
+    - array
```

Comment:
Oh ok! What is not transparent at all uff. It would be cooler if they had package `alex` which then imports what they need hm. Is it possible to put a comment here mentioning that Alex needs it? or we just rely on compiler error warning people if they try to remove this?